common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -99,6 +99,7 @@
     private static final AtomicLong DIRECT_MEMORY_COUNTER;
     private static final long DIRECT_MEMORY_LIMIT;
     private static final ThreadLocalRandomProvider RANDOM_PROVIDER;
+    private static final boolean TRY_ALLOCATE_UNINITIALIZED_ARRAY;
```

Comment:
can we instead make this an `int` which indicates the size to start allocating using unsafe? Something like `<0` means don't use unsafe, `<=0` means use unsafe for arrays of this size or greater. Default of 512b or 1024b based upon the benchmarks sgtm.